.github/workflows/automated-tests.yml

```
@@ -0,0 +1,16 @@
+name: Build Figma Tokens to CSS
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run test
```

```suggestion
      - run: npm run test
```
Je ne vois pas de scripts `ci` dans kitten

---

C'est une commande interne à `npm` pour insaller directement depuis le package lock, sans le modifier. C'est spécifique pour les env de test, de ci, etc : https://docs.npmjs.com/cli/v8/commands/npm-ci